sdk/storage/azure-storage-file-datalake/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
 ## 12.1.0-beta.1 (Unreleased)
-
+- Fixed a NPE caused due to deserializing a non existant lastModifiedTime.
```

Comment:
Was this actually done in this PR? I'm not really seeing a change that would indicate this.

Reply:
No, I did it in a PR earlier this week, just forgot to update Changelog